.github/workflows/asf-updates.yml

```diff
@@ -0,0 +1,86 @@
+name: Update ASF APIs
+on:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
I'm wondering whether we really need to do this on a daily basis. currently we wouldn't even have the capacity to react quickly enough to changes in the API.
is there a particular argument for the daily schedule, or would, say, weekly be enough?

open to discuss

Reply:
Yeah, I also think weekly would be enough. I'll change it to Monday morning (similar to RenovateBot).